internal/runners/artifacts/artifacts.go

```diff
@@ -201,14 +207,19 @@ func (b *Artifacts) Run(params *Params) (rerr error) {
 }
 
 func (b *Artifacts) outputPlain(out *StructuredOutput, fullID bool) error {
-	if out.HasFailedArtifacts {
-		b.out.Error(locale.T("warn_has_failed_artifacts"))
-	}
-
 	for _, platform := range out.Platforms {
 		b.out.Print(fmt.Sprintf("• [NOTICE]%s[/RESET]", platform.Name))
 		for _, artifact := range platform.Artifacts {
-			if artifact.URL == "" {
+			switch {
+			case len(artifact.Errors) > 0:
+				b.out.Print(fmt.Sprintf("  • %s ([ERROR]%s[/RESET])", artifact.Name, locale.T("artifact_status_failed")))
+				b.out.Print(fmt.Sprintf("    ├─ %s: [ERROR]%s[/RESET]", locale.T("artifact_status_failed_message"), strings.Join(artifact.Errors, ": ")))
+				b.out.Print(fmt.Sprintf("    └─ %s: [ACTIONABLE]%s[/RESET]", locale.T("artifact_status_failed_log"), artifact.LogURL))
+				continue
+			case artifact.status == types.ArtifactSkipped:
+				b.out.Print(fmt.Sprintf("  • %s ([NOTICE]%s[/RESET])", artifact.Name, locale.T("artifact_status_skipped")))
+				continue
```

Comment:
I added this because for the project I tested with, one artifact had a status of "SKIPPED" that was being shown as "Building ...", which was not accurate. Even though this wasn't in the mock or ACs, I went ahead and made things change.